static/javascripts/script.js

```diff
@@ -306,19 +306,27 @@
 		$item.appendTo('#members');
 	}
 
-	function addMembers(){
-		$.getJSON('https://api.github.com/orgs/cloudflare/members?callback=?', function (result) {
+	function addMembers( count, url ){
+		$.getJSON(url + '&callback=?', function (result) {
```

Comment:
When the `callback=?` search parameter is specified, the response is:
```json
{
  "message": "Invalid callback: \"?\"",
  "documentation_url": "https://developer.github.com/v3/#json-p-callbacks"
}
```
To fix this, remove the `callback` search parameter.

Reply:
@ExE-Boss jQuery replaces the question mark with a valid paramater.